internal/service/service.go

```diff
@@ -0,0 +1,146 @@
+package service
+
+import (
+	"errors"
+	"github.com/VTerenya/employees/internal"
+	"github.com/VTerenya/employees/internal/repository"
+)
+
+type Handler interface {
```

Comment:
Please use different names for interfaces. You have 3 `Handler` interfaces in 3 packages.
The name of interface should reflect the behavior which it represents.

Reply:
+